index.html

```diff
@@ -36,7 +36,158 @@ <h2>
             <div>
                <img src="Images/Social Media.png" alt="Social icons">
             </div>
-        
+        </section>
+        <section class="works">
+            <div class="content1">
+                <h1>
+                    My Recent Works
+                </h1>
+                <img src="Images/Img Placeholder.png" alt="">
+                <h2>
+                    Multi-Post Stories
+                </h2>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s,
+                     when an unknown printer took a standard dummy text.
+                </p>
+                <div>
+                    <h3>css</h3>
+                    <h3>html</h3>
+                    <h3>bootstrap</h3>
+                    <h3>Ruby</h3>
+                </div>
+                <button>See Project</button>
+            </div>
+            </div>
+            <div class="cards">
+                <h1>
+                    Profesional Art Printing Data
+                </h1>
```

Comment:
- [ ] Kindly I recommend to you use  a header tag `hx` for  project title (e.g., "Profesional Art Printing Data") instead of `h1` tag ,This way will be semantically correct and the screen readers and other technologies will thank you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.
- NB: There should be just one h1 in an HTML file, for SEO(Search Engine Optimization) and web accessibility purposes.